Contents.swift

```diff
@@ -68,6 +69,11 @@ class ChatCell: UITableViewCell {
     }
 }
 
+extension ChatCell {
+    override var textLabel: UILabel? { return messageLabel }
+    override var imageView: UIImageView? { return iconImageView }
```

Comment:
I agree that this prevents users from accessing the wrong properties, but typically if you’re using a subclass cell you’d just use the properties appropriately